### ISSUE TITLE
target-redshift to SDK

### ISSUE THREAD

**Issue by aaronsteers** [MeltanoLabs/Singer-Most-Wanted#18]
Possible inspiration / starting point: https://github.com/transferwise/pipelinewise-target-redshift

**Comment by techtangents** [comment]
At my work, we have some huge tables in postgres that we want to ingest into Redshift. There are a few around 5 million rows, a couple are around 50 million, and one is about 5 billion. 

The batch messages would speed this up dramatically. At this stage, ingesting several million rows at a time is error-prone for us, so the initial ingestions of tables like this are troublesome. The 5 billion row table is infeasible right now.

**Comment by pnadolny13** [comment]
Theres been a couple updates in slack related to this:
- @menzenski was looking for example SQL targets to get a sense of the difficulty level to rewrite it https://meltano.slack.com/archives/C01RKUVUG4S/p1675462154066899
- JP (slack user) requesting info on how to move forward with the transferwise variant that is not maintained well https://meltano.slack.com/archives/C013EKWA2Q1/p1675861468111389

**Comment by menzenski** [comment]
Yeah, I have been challenged by the lack of maintainer response on the transferwise variant also. I've had an open PR there for a while and there's been no activity in something like 18 months it seems.

**Comment by pnadolny13** [comment]
More updates in slack https://meltano.slack.com/archives/C013Z450LCD/p1684388483962929.

A few new forks and a user looking to take over maintenance of the default target.
